src/kt_class.h

```diff
@@ -7,6 +7,10 @@
 #include "kt_function.h"
 #include "kt_signal_info.h"
 #include "java_instance_wrapper.h"
+#include "kt_constructor.h"
+
+const int CONSTRUCTOR_MAX_ARGS_SIZE = 5;
+const int MAX_CONSTRUCTOR_SIZE = CONSTRUCTOR_MAX_ARGS_SIZE + 1;
```

Comment:
Maybe add a comment on where to change the same constant on the jvm so it doesn't get forgotten if changed at either the cpp or the jvm side

Reply:
done